### PR TITLE
Gui test 913

### PR DESCRIPTION
**Pull request by maryvictol**
This PR add testcase and e2e test for https://github.com/epam/cloud-pipeline/pull/913

### CODE REVIEW

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/LaunchParametersTest.java [hunk 1]
```diff
@@ -80,14 +87,21 @@ public class LaunchParametersTest extends AbstractSeveralPipelineRunningTest
...
     private final String registry = C.DEFAULT_REGISTRY;
     private final String group = C.DEFAULT_GROUP;
     private final String tool = C.TESTING_TOOL_NAME;
+    private final String customTag = "test_tag";
+    private int[] scaling = new int[4];
+    private int[] sizeDisk = new int[4];
     private String initialLaunchSystemParameters;
 
+
+
```
Comment (kamyshova) [inline e2e/gui/src/test/java/com/epam/pipeline/autotests/LaunchParametersTest.java]: Clean up, please

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/LaunchParametersTest.java [hunk 2]
```diff
@@ -409,7 +434,7 @@ public void allowToUseR6iInstanceFamilyInSGEhybridAutoscaling() {
         try {
             logoutIfNeeded();
             loginAs(admin);
-            setUserSettings("*");
+            setUserSettings("r6i.*");
```
Comment (kamyshova) [inline e2e/gui/src/test/java/com/epam/pipeline/autotests/LaunchParametersTest.java]: Lets introduce a constant

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/LaunchParametersTest.java [hunk 3]
```diff
@@ -447,6 +472,72 @@ public void allowToUseR6iInstanceFamilyInSGEhybridAutoscaling() {
...
+                                           .waitUntilTextAppears(getLastRunId())
+                                           .execute(command)
+                                           .assertNextStringIsVisible(command, rootHost)
+                                           .lastCommandResult(command);
+                                       scaling = diskSize(lastResult);
+                                       sizeDisk[0] = (int)Math.floor(scaling[0] * 1.5);
+                                       for(int i = 0; i < 3; i++) {
+                                           sizeDisk[i+1] = (int)Math.floor(sizeDisk[i] * 1.5);
+                                       }
+                                       shell.execute(format("fallocate -l %sG test.big", scaling[2] - 1));
```
Comment (kamyshova) [inline e2e/gui/src/test/java/com/epam/pipeline/autotests/LaunchParametersTest.java]: Let's extract to method. Please, add missing spaces.

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/LaunchParametersTest.java [hunk 4]
```diff
@@ -447,6 +472,72 @@ public void allowToUseR6iInstanceFamilyInSGEhybridAutoscaling() {
...
+                                   .ssh(shell -> {
+                                       String lastResult = shell
+                                           .waitUntilTextAppears(getLastRunId())
+                                           .execute(command)
+                                           .assertNextStringIsVisible(command, rootHost)
+                                           .lastCommandResult(command);
+                                       scaling = diskSize(lastResult);
+                                       sizeDisk[0] = (int)Math.floor(scaling[0] * 1.5);
+                                       for(int i = 0; i < 3; i++) {
+                                           sizeDisk[i+1] = (int)Math.floor(sizeDisk[i] * 1.5);
```
Comment (kamyshova) [inline e2e/gui/src/test/java/com/epam/pipeline/autotests/LaunchParametersTest.java]: Let's extract '1.5' magic number to constant

**Review by kamyshova — CHANGES_REQUESTED** [review]

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/LaunchParametersTest.java (hunk 4 above)
Comment (maryvictol) [reply e2e/gui/src/test/java/com/epam/pipeline/autotests/LaunchParametersTest.java]: DONE

**Review by kamyshova — APPROVED** [review]

File: docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/913.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
+# Add support for autoscaling filesystem for AWS
+
+Test verifies that for all runs with the file system autoscaling capability being enabled the daemon checks current filesystem usage and attach additional disks in case there is no enough space in the filesystem.
+
+**Prerequisites**:
+- Admin user
+
```
Comment (NShaforostov) [inline docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/913.md:7]: Probably, we should add items to the prerequisites - at least, to check that:

- system preference `cluster.instance.hdd.scale.enabled` is enabled
- system preference `cluster.instance.hdd.scale.delta.ratio` is 0.5

File: docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/913.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+| 9 | Click **Launch** button | |
+| 10 | At the Runs page, click the just-launched run | |
+| 11 | Wait until the SSH hyperlink appears | |
+| 12 | Click the SSH hyperlink | |
+| 13 | In the opened tab, enter and perform the command `df -hT` | File system space usage table is shown in the response |
+| 14 | Store ***Avail*** value for filesystem with type `btrfs` | |
+| 15 | Enter and perform the command <br> `fallocate -l <file_size>G test.big`, <br> where `<file_size>` is a value equals to the value kept at the step 14 minus 1 | 
+| 16 | Return to the run log page | |
+| 17 | Wait until the ***FilesystemAutoscaling*** task appears | |
+| 18 | Click ***FilesystemAutoscaling*** task | Log contains record <br> `Filesystem /ebs was autoscaled <value1>G + <value2>G = <value3>G.`, <br> where <br> `<value1>` is value kept in the step 14, <br> `<value2>` is `<value1> * 0.5`, <br> `<value3> = <value1> + <value2>` |
```
Comment (NShaforostov) [inline docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/913.md:27]: `<value1>` in this step shall be equal to the **Size** value for filesystem with type `btrfs` in output of step 13 (not **Avail** value)

**Review by NShaforostov — APPROVED** [review]
Please, take into account notes from the comments before the merge.

**Pull request merged**